web/src/components/Table/Table.tsx

```diff
@@ -0,0 +1,434 @@
+        )
+      })}
+      <td>
+        <div className="row-link-menu">
+          <a>
+            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="black" className="bi bi-box-arrow-up-right" viewBox="0 0 16 16">
+              <path fillRule="evenodd" d="M8.636 3.5a.5.5 0 0 0-.5-.5H1.5A1.5 1.5 0 0 0 0 4.5v10A1.5 1.5 0 0 0 1.5 16h10a1.5 1.5 0 0 0 1.5-1.5V7.864a.5.5 0 0 0-1 0V14.5a.5.5 0 0 1-.5.5h-10a.5.5 0 0 1-.5-.5v-10a.5.5 0 0 1 .5-.5h6.636a.5.5 0 0 0 .5-.5"/>
+              <path fillRule="evenodd" d="M16 .5a.5.5 0 0 0-.5-.5h-5a.5.5 0 0 0 0 1h3.793L6.146 9.146a.5.5 0 1 0 .708.708L15 1.707V5.5a.5.5 0 0 0 1 0z"/>
+            </svg>
+          </a>
+        </div>
+      </td>
+      <td>
+        <span className="row-qr-btn">
+          <button onClick={() => {
+            setShowQR(true); 
+            setQRURL(createShareURL(props));
+            }}>
+            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" className="bi bi-qr-code" viewBox="0 0 16 16">
+              <path d="M2 2h2v2H2z"/>
+              <path d="M6 0v6H0V0zM5 1H1v4h4zM4 12H2v2h2z"/>
+              <path d="M6 10v6H0v-6zm-5 1v4h4v-4zm11-9h2v2h-2z"/>
+              <path d="M10 0v6h6V0zm5 1v4h-4V1zM8 1V0h1v2H8v2H7V1zm0 5V4h1v2zM6 8V7h1V6h1v2h1V7h5v1h-4v1H7V8zm0 0v1H2V8H1v1H0V7h3v1zm10 1h-1V7h1zm-1 0h-1v2h2v-1h-1zm-4 0h2v1h-1v1h-1zm2 3v-1h-1v1h-1v1H9v1h3v-2zm0 0h3v1h-2v1h-1zm-4-1v1h1v-2H7v1z"/>
+              <path d="M7 12h1v3h4v1H7zm9 2v2h-3v-1h2v-1z"/>
+            </svg>
+          </button>
+        </span>
+        <span className="row-clipboard-btn">
+          <button onClick={() => copyToClipboard(props)}>
+            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" className="bi bi-clipboard" viewBox="0 0 16 16">
+              <path d="M4 1.5H3a2 2 0 0 0-2 2V14a2 2 0 0 0 2 2h10a2 2 0 0 0 2-2V3.5a2 2 0 0 0-2-2h-1v1h1a1 1 0 0 1 1 1V14a1 1 0 0 1-1 1H3a1 1 0 0 1-1-1V3.5a1 1 0 0 1 1-1h1z"/>
+              <path d="M9.5 1a.5.5 0 0 1 .5.5v1a.5.5 0 0 1-.5.5h-3a.5.5 0 0 1-.5-.5v-1a.5.5 0 0 1 .5-.5zm-3-1A1.5 1.5 0 0 0 5 1.5v1A1.5 1.5 0 0 0 6.5 4h3A1.5 1.5 0 0 0 11 2.5v-1A1.5 1.5 0 0 0 9.5 0z"/>
+            </svg>
+          </button>
+        </span>
+      </td>
+    </tr>
+  );
+}
+
+export default function Table({ data, currentPage, rowCountPerPage, visibleColumns, hideColumn, showHistogram, setShowQR, setQRURL, filters, setFilters, filterPopupRef, showLineNumbers }: TableProps) {
+  const [sortConfig, setSortConfig] = useState<{ key: keyof TableRowProps; direction: 'ascending' | 'descending' } | null>(null);
+  const [filterColumn, setFilterColumn] = useState<string | null>(null);
+  const [showFilterPopup, setShowFilterPopup] = useState<boolean>(false);
+  const [anchorRefs, setAnchorRefs] = useState<{ [key: string]: HTMLDivElement | null }>({});
+  const [filterType, setFilterType] = useState('')
+
+
+  const rowsPerPage = rowCountPerPage;
```

Comment:
Why exactly is this alias created?

Reply:
It is an unnecessary variable that i forgot to delete while creating table at first it was defined as rowsPerPage = 25 so it is not necessary now i'll make the change.